staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/mutating/dispatcher.go

```diff
@@ -40,6 +41,13 @@ import (
 	"k8s.io/apiserver/pkg/util/webhook"
 )
 
+func init() {
+	// DefaultMaxRequestBytes is the default maximum client request size in
+	// bytes the etcd server will accept. Thus, we set the same limit on
+	// the size increase the "copy" operations in a json patch can cause.
+	jsonpatch.AccumulatedCopySizeLimit = int64(embed.DefaultMaxRequestBytes)
```

Comment:
this is only the default value of a tuneable etcd parameter. I wouldn't recommend setting our limit lower than the limit at which etcd starts warning the server operator:

https://github.com/kubernetes/kubernetes/blob/9d6ebf6c78f406d8639aae189901e47562418071/vendor/github.com/coreos/etcd/etcdserver/server.go#L270-L272

https://github.com/kubernetes/kubernetes/blob/9d6ebf6c78f406d8639aae189901e47562418071/vendor/github.com/coreos/etcd/etcdserver/server.go#L90

Reply:
Good point. Updated.